test/resources/microshift-process.resource

```diff
@@ -20,3 +22,42 @@ MicroShift Version
     Should Not Be Empty    ${version_text}
     ${version}=  Yaml Parse  ${version_text}
     RETURN  ${version}
+
+MicroShift Is Ready
+    [Documentation]  Check the /readyz endpoint
+    ${stdout}=  Run With Kubeconfig  oc get --raw='/readyz'
+    Should Be Equal As Strings  ${stdout}  ok  strip_spaces=True
+
+MicroShift Is Live
+    [Documentation]  Check the /livez endpoint
+    ${stdout}=  Run With Kubeconfig  oc get --raw='/livez'
+    Should Be Equal As Strings  ${stdout}  ok  strip_spaces=True
+
+Wait For MicroShift
+    [Documentation]  Wait for various checks to ensure MicroShift is online.
+    Wait Until Keyword Succeeds
+    ...  30x
+    ...  10s
+    ...  MicroShift Is Ready
+    Wait Until Keyword Succeeds
+    ...  30x
+    ...  10s
+    ...  MicroShift Is Live
+    # We could also wait for relevant pods. Can we restructure the
```

Comment:
Actually it does work even if not in ostree. You will see an error about not being in an ostree host but thats only cosmetic. Checks run and return codes reflect situation. Needs some refactoring though, as it always expects topolvm to be up, even if there is no lvmd.